code/nodes/saveGraph/saveGraph.q

```diff
@@ -1,6 +1,24 @@
-// Save all the graphs relevant for the generation of reports and for prosperity
 \d .automl
 
+// Save all the graphs relevant for the generation of reports and for prosperity
+
+// @kind function
+// @category node
+// @fileoverview Save all graphs needed for reports 
+// @param params {dict} All data generated during the preprocessing and
+//  prediction stages
+// @return {null} All graphs needed for reports are saved to appropriate location
+saveGraph.node.function:{[params]
+  savePath:params[`pathDict;`images];
```

Comment:
Check for `saveOpt in 1 2` which is in each of the functions happens in each of the functions called below can be called once initially and just exit the function at this point returning params